web/bananojs-hw.js

```diff
@@ -83,13 +226,13 @@ window.bananocoin.bananojsHw.getLedgerAccountSigner = async (accountIx) => {
         hwBlockData.representative = blockData.representative;
         hwBlockData.balance = blockData.balance;
         hwBlockData.recipient = window.bananocoinBananojs.getBananoAccount(
-            blockData.link,
+          blockData.link,
         );
 
         const cacheBlockData = {};
         const cacheBlocks = await bananodeApi.getBlocks(
-            [blockData.previous],
-            true,
+          [blockData.previous],
```

Comment:
One day I'll add prettier to this package to prevent indentation all over the place.